app/registries/foreman/access_permissions.rb

```diff
@@ -18,6 +18,18 @@
       :bookmarks => [:index, :show, :auto_complete_search, :welcome],
       :"api/v2/bookmarks" => [:index, :show],
     }, :public => true
+    # file URLs are short-lived, hence can be public
```

Comment:
Will this be true for every plugin? This feels like something that's hard to change in the future and requires some thought.

Reply:
The idea is that the URL will be given only to authorized user. Once the URL is out, by default I think it can be public. If we want to change it, we will need to customize the built in controllers.

Reply:
So maybe I don't quite understand what public does.

Reply:
ActiveStorage generates a URL for each file it needs to serve. This URL is unique and served by either `blob/proxy` or by `blob/redirect` controller. Those controllers are generic - no matter what kind of files they are serving.
The idea is that the URL for the file will appear on a page only for authorized users.
Example:
Let's say I have a page with a button to download a report. This page is properly authorized (only admin will see the download button). The download button leads to `/blob/proxy/blob_123`. This new URL is not easily authorized, since we don't know what kind of file we are serving this time.